app/keepers/keepers.go

```diff
@@ -149,6 +168,8 @@ type AppKeepers struct {
 	ValidatorSetPreferenceKeeper *valsetpref.Keeper
 	ConcentratedLiquidityKeeper  *concentratedliquidity.Keeper
 	CosmwasmPoolKeeper           *cosmwasmpool.Keeper
+	GroupKeeper                  *groupkeeper.Keeper
+	NFTKeeper                    *nftkeeper.Keeper
```

Comment:
Why did we add these modules? Lets remove them?

Reply:
Removed https://github.com/osmosis-labs/osmosis/pull/6413/commits/9cd9cdba26e611bfe866e329404de902fa8d357d